### DISCUSSION TITLE
Welcome to CAN Discussions!

### DISCUSSION THREAD

**Discussion by nberlette** [nberlette/canbus#1] [Announcements]
## Welcome!

We just fired up the brand new GitHub Discussions platform, and we'd love to see what you have to share with the community. Please start a discussion here if you have:

- Questions or Comments about the [canbus wiki](/nberlette/canbus/wiki) or [repository](/nberlette/canbus)
- Ideas for a new project or feature you'd like to be developed
- Projects you've made that you want to share with the community
- Literally anything at all to get some actual discussion started here  👀 

Thanks for stopping by!

**Comment by MatinF** [comment]
Hi Nicholas,

Thanks for sharing a nice repository here - and for sharing the CSS Electronics OBD2 DBC file. 

I would recommend to remove the J1939 DBC file, however, from the repository as it is illegal to share this in public (it requires a legal license from SAE to use). The file can be purchased below in a fully legal version and with the latest updates:
https://www.csselectronics.com/products/j1939-dbc-file-pgn-spn

Appreciate your work and support!
